src/components/DonationItemsTable/DonationItemsTable.jsx

```diff
@@ -44,44 +49,106 @@ const DonationItemsTable = () => {
 const DonationItems = ({ category }) => {
   const [data, setData] = useState([]);
   const { backend } = useBackend();
+  const [currentItemNum, setCurrentItemNum] = useState(0);
+  const [currentPageNum, setCurrentPageNum] = useState(1);
+  const [pageLimit, setPageLimit] = useState(1);
+  const [selectedItem, setSelectedItem] = useState(null);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
   const TABLE_HEADERS = ['Id', 'Name', 'Quantity Type', 'Price', 'Category'];
+
   useEffect(() => {
-    const getData = async () => {
-      try {
-        const response = await backend.get(`/value/filter/${category}`);
-        setData(response.data);
-      } catch (error) {
-        console.error('Error getting donation items:', error);
-        setData([]);
-      }
-    };
-    getData();
-  }, [backend, category]);
+    loadInfo();
+  }, [currentPageNum, category]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'loadInfo'. Either include it or remove the dependency array.